osbenchmark/client.py

```diff
@@ -156,8 +155,9 @@ def __init__(self, hosts, client_options):
                 # advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings"
                 urllib3.disable_warnings()
             else:
+                # check_hostname should be False if any host is an IP address
```

Comment:
Please clarify the comment a little more -- "the peer's hostname can be matched only if it is provided, i.e.,  hostname checking is disabled if IP addresses are supplied instead of hostnames.  An exception is raised if the list of hosts is a mix of both hostnames and IP addresses.

Also consider whether a warning would be a better option here rather than terminating with an exception.  Something like: "Although certificate verification is enabled, peer hostnames will not be matched since the host list is a mix of names and IP addresses".

Reply:
Thanks for the tip! I agree that swapping out the exception with a warning is a better option since the only thing users are forgoing is the verification that OSB client is talking to the correct server. I'll update this.